firmware/fsm.c

```diff
@@ -1135,7 +1154,7 @@ void fsm_msgNEMGetAddress(NEMGetAddress *msg)
 
 		bool qrcode = false;
 		for (;;) {
-			layoutAddress(resp->address, desc, qrcode);
+			layoutAddress(resp->address, desc, qrcode, false);
```

Comment:
@saleemrashid If NEM only uses upper case/numbers/hyphen in address, we can change this to true to get QR codes with better error correction level.

Reply:
ACK (it's Base32)

Reply:
Applied in 892bb8501a1282535a9c721d0a3187f51aa5585b